src/Microsoft.Kusto.ServiceLayer/DataSource/IKustoClient.cs

```diff
@@ -2,17 +2,11 @@
 using System.Data;
 using System.Threading;
 using System.Threading.Tasks;
-using Kusto.Language;
 
 namespace Microsoft.Kusto.ServiceLayer.DataSource
 {
     public interface IKustoClient
     {
-        /// <summary>
-        /// SchemaState used for getting intellisense info.
-        /// </summary>
-        GlobalState SchemaState { get; }
-
```

Comment:
Moved to KustoIntellisenseClient. This was done to allow unit testing of this file. Previously, any cluster URL passed in would at tempt to initialize the intellisense.